File: cs50/week0to5-c/pset1/mario/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    //get user input from 1 to 8
    int n;
    do
    {
        n = get_int("Height: ");
    }
    while (n < 1 || n > 8);


    // print the pyramid
    for (int i = 1; i <= n; i++)
    {
        // printing the first pyramid
        for (int j = n - i; j > 0; j--)
        {
            printf(" ");
        }
        for (int k = 0; k < i; k++)
        {
            printf("#");
        }

        // two spaces between the pyramids
        printf("  ");

        // printing the second pyramid
        for (int k = 0; k < i; k++)
        {
            printf("#");
        }

        printf("\n");

    }

}

File: cs50/week6to10-python/pset6/mario/less/mario.py
from cs50 import get_int


def main():
    # get integer input from the user
    while True:
        user_choice = get_int("Height: ")
        if user_choice < 9 and user_choice > 0:
            break

    # call the print pyramid function to print the pyramid
    print_pyramid(user_choice)


def print_pyramid(number):
    '''
    prints a pyramid for mario to climb
    input: integer number between 1 to 8
    output: pyramid printed to console
    '''

    for i in range(number):
        for j in range(number - (i+1)):
            print(" ", end="")
        print("#" * (i+1))


if __name__ == "__main__":
    main()